spec/classes/dns_init_spec.rb

```diff
@@ -365,6 +365,21 @@
         it { should contain_file("#{etc_directory}/dns-key.key") }
       end
 
+      describe 'with config_check set to false' do
+        let :params do
```

Comment:
For consistency with the other tests, maybe
`let(:params) { {:config_check => false} }`??